resources/views/battle-results.blade.php

```diff
@@ -68,4 +69,17 @@
             margin-bottom: 10px;
         }
     </style>
-@endsection
\ No newline at end of file
+@endsection
+
+@section("scripts")
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jQuery-slimScroll/1.3.8/jquery.slimscroll.min.js"></script>
+    <script type="text/javascript">
+        $(function () {
+            $('#battle-results').slimScroll({
+                height: (73 * 5) + 'px',
```

Comment:
It's minimum height, like when we add the 5th panel, the scroll bar will shown, understand?